### ISSUE TITLE
Rewritten is acting like it's not installed on Firefox & Nightly.

### ISSUE THREAD

**Issue by Milapurr** [AprilSylph/XKit-Rewritten#1273]
### Platform

Windows 10

### Browser

Firefox Stable: 117.0 (64-bit), Firefox Nightly: 119.0a1 (2023-09-08) (64-bit)

### Addon version

v0.22.1 (beta)

### Details

**Firefox Stable**
As of 2:30 AM EST:

- Trim Reblogs no longer works

(Trim reblogs while in drafts does not have the scissors and other options)

- Quick Tags no longer works

(Tag icon doesn't show up)
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/144516935/16e4bb9e-d67e-4102-bfd3-141f607d268b)

- Quick Reblog no longer works

(Hovering over reblog doesn't drop down the options to: queue, draft, reblog)
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/144516935/08e75cd5-c2b3-4fc7-9627-60bed9b80efc)

- Mutual Checker no longer works
(Will sometimes load if you scroll down the dash)

![image](https://github.com/AprilSylph/XKit-Rewritten/assets/144516935/a89354de-d254-4ab0-8258-27c4cfcb0191)
(this is a mutual, followers has them as a mutual)

- Tweaks no longer works
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/144516935/92f542c4-771c-4b15-acad-e8263985fd6c)

(Evident by the carousel not being turned into a line)
(All in all, it acts like Xkit-Rewritten was never installed. I did a full uninstall and reinstall of the extension)

**- Firefox Nightly:**

- Mutual Checker no longer works

(Icon no longer appears next to mutuals despite being mutuals)
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/144516935/87931661-fcb0-4464-a7a1-1371c13ddc99)
(same thing, mutual but not showing up as one.

If this is any help: **Chrome Version 116.0.5845.180 (Official Build) (64-bit)** & **Opera One(version: 101.0.4843.43)** are unaffected.

I could be missing some.

As of 3:50 AM, sometimes refreshing the page will work. Sometimes not. Intermittent issue?

**Comment by marcustyphoon** [comment]
Haven't looked into this yet, but I have to say: great, comprehensive bug report!

**Comment by AprilSylph** [comment]
Unable to reproduce over 10 reloads on Windows 10, Firefox 117.0 (64-bit) using XKit Rewritten 0.22.1

**Comment by Milapurr** [comment]
Whatever happened might have fixed itself. I know Tumblr's been acting up most of the night on my end. I'm able to reload just fine as of 11:44 AM.

**Comment by enchanted-sword** [comment]
Sorry, this one's on me. I was trying to solve a annoying race condition issue with my script and I only this morning realized that the solution I had applied a) affected normal page reloads as well as the background page loads I was targeting and b) the solution was more hacky than I thought it was, and sometimes prevented XKit from working on affected pages. Now that I've actually gotten some sleep, I found a _far_ better permanent solution and updated just now.
